package.json

```diff
@@ -129,6 +129,8 @@
     "electron:packager-win": "electron-packager .package bitburner --platform win32 --arch x64 --out .build --overwrite --icon .package/icon.png",
     "electron:packager-mac": "electron-packager .package bitburner --platform darwin --arch x64 --out .build --overwrite --icon .package/icon.png",
     "electron:packager-linux": "electron-packager .package bitburner --platform linux --arch x64 --out .build --overwrite --icon .package/icon.png",
-    "allbuild": "npm run build && npm run electron && git add --all && git commit -m \"allbuild commit $(which git)\" && git push -f -u origin dev"
+    "preversion": "npm install && npm run test",
+    "version": "sh ./tools/build-release.sh && git add --all",
+    "postversion": "git push -u origin dev && git push --tags"
```

Comment:
IMHO `git add --all` from the command above should be here

Reply:
npm creates the commit right at the end version, but before running postversion. 
So the git add --all needs to be in the "version" hook otherwise the commit won't contain the files.